jovian/tests/utils/test_api.py

```diff
@@ -338,6 +340,8 @@ def test_create_gist_simple_no_gist_slug(mock_requests_post):
             version_title="first version",
         )
 
+        assert capsys.readouterr().err.strip() == '[jovian] Error: Uploaded gist has a warning'.strip()
```

Comment:
Instead of checking for the warning in all the tests, it would be better to create a specific test for that.